Automattic/wp-calypso
# Activity Logs: Add rewind status http handler

Add rewind status actions and http request for the Jetpack Activity Log & Rewind feature.

Breaking down proof-of-concept PR #10862 into merge-able pieces.

This PR supersedes #14647 using the data layer instead of undocumented.js, starting with a rewind status request.

**To Test**
* Fire the action in the console, for example:
`dispatch( { type: 'REWIND_STATUS_REQUEST', siteId: 'sseear.wpsandbox.me' } )`
* Check that the network request is made in the _Network_ tab
* (also includes unit tests)
